### PR TITLE
New package: Aurelitec.Thumbico version 1.5.0.25

### PR DESCRIPTION
**Pull request by RalfEs73**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/148028)

### CODE REVIEW

**Review by stephengillie — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Verify: 
image

File: manifests/a/Aurelitec/Thumbico/1.5.0.25/Aurelitec.Thumbico.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Aurelitec.Thumbico
+PackageVersion: 1.5.0.25
+PackageLocale: en-US
+Publisher: Aurelitec
+PackageName: Thumbico
+License: MIT-Lizenz
```
Comment (Trenly) [inline manifests/a/Aurelitec/Thumbico/1.5.0.25/Aurelitec.Thumbico.locale.en-US.yaml:9]:
```suggestion
License: MIT
```

File: manifests/a/Aurelitec/Thumbico/1.5.0.25/Aurelitec.Thumbico.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Aurelitec.Thumbico
+PackageVersion: 1.5.0.25
+PackageLocale: en-US
+Publisher: Aurelitec
+PackageName: Thumbico
+License: MIT-Lizenz
+Copyright: Copyright © 2011-2015 Aurelitec (http://www.aurelitec.com)
+ShortDescription: Thumbico Installer
```
Comment (Trenly) [inline manifests/a/Aurelitec/Thumbico/1.5.0.25/Aurelitec.Thumbico.locale.en-US.yaml:11]:
```suggestion
ShortDescription: File icons and thumbnails of any size
```

File: manifests/a/Aurelitec/Thumbico/1.5.0.25/Aurelitec.Thumbico.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Aurelitec.Thumbico
+PackageVersion: 1.5.0.25
+PackageLocale: en-US
+Publisher: Aurelitec
+PackageName: Thumbico
```
Comment (Trenly) [inline manifests/a/Aurelitec/Thumbico/1.5.0.25/Aurelitec.Thumbico.locale.en-US.yaml:8]:
```suggestion
PackageName: Thumbico
PackageUrl: https://www.aurelitec.com/thumbico/
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Pull request closed without merging**